verilog/tools/formatter/verilog_format.cc

```diff
@@ -240,8 +241,20 @@ int main(int argc, char** argv) {
     return 1;
   }
 
-  if (!lines_to_format.empty() && file_args.size() > 2) {
-    std::cerr << "Note, applying --lines rules to all files" << std::endl;
+  // Some sanity checks if multiple files are given.
+  if (file_args.size() > 2) {
+    if (!lines_to_format.empty()) {
+      std::cerr << "Providing --lines but gave multiple different files."
```

Comment:
suggest: "--lines=... only works for a single file"

We should also update the help string for this flag similarly.

Reply:
Sounds better.done.